scripts/resource/README.md

```diff
@@ -0,0 +1,41 @@
+# Resource class generator
+
+## prerequisites
+
+- [poetry](https://python-poetry.org/)
+- [oc](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/) or [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- Kubernetes/Openshift cluster
+
+## Preparation
+
+###### Generate explain file
+
+```bash
+oc explain deployment --recursive > deployment.txt # Where deployment can be any KIND in the cluster
+
+```
+
+###### Check if the resource is namespaced or not
+
+```bash
+oc api-resources --namespaced | grep deployment | wc -l # Where deployment is the same KIND from `oc explain` command
```

Comment:
This command won't always work. e.g CDI. kind is `CDI` and it's not namespaced. But 
```
(openshift-python-wrapper-py3.12) dbasunag@dbasunag-mac openshift-python-wrapper % oc api-resources --namespaced | grep cdi            
dataimportcrons                       dic,dics                                                                               cdi.kubevirt.io/v1beta1                       true         DataImportCron
datasources                           das                                                                                    cdi.kubevirt.io/v1beta1                       true         DataSource
datavolumes                           dv,dvs                                                                                 cdi.kubevirt.io/v1beta1                       true         DataVolume
volumeclonesources                                                                                                           cdi.kubevirt.io/v1beta1                       true         VolumeCloneSource
volumeimportsources                                                                                                          cdi.kubevirt.io/v1beta1                       true         VolumeImportSource
volumeuploadsources                                                                                                          cdi.kubevirt.io/v1beta1                       true         VolumeUploadSource
openstackvolumepopulators             osvp,osvps                                                                             forklift.cdi.kubevirt.io/v1beta1              true         OpenstackVolumePopulator
ovirtvolumepopulators                 ovvp,ovvps                                                                             forklift.cdi.kubevirt.io/v1beta1              true         OvirtVolumePopulator
uploadtokenrequests                   utr,utrs                                                                               upload.cdi.kubevirt.io/v1beta1                true         UploadTokenRequest
(openshift-python-wrapper-py3.12) dbasunag@dbasunag-mac openshift-python-wrapper % 
(openshift-python-wrapper-py3.12) dbasunag@dbasunag-mac openshift-python-wrapper % oc api-resources --namespaced | grep cdi | wc -l
       9
(openshift-python-wrapper-py3.12) dbasunag@dbasunag-mac openshift-python-wrapper %
```
and for kubevirt it is:
```
(openshift-python-wrapper-py3.12) dbasunag@dbasunag-mac openshift-python-wrapper % oc api-resources --namespaced | grep kubevirt | wc -l      
      25
(openshift-python-wrapper-py3.12) dbasunag@dbasunag-mac openshift-python-wrapper % 
```
But `oc api-resources  --namespaced | awk ' { print $1 }' | grep cdi | wc -l` would work always.

Reply:
Fixed